src/fq_default_poly/test/t-evaluate_fq_default.c

```diff
@@ -0,0 +1,122 @@
+/*
+    Copyright (C) 2021 William Hart
+
+    This file is part of FLINT.
+
+    FLINT is free software: you can redistribute it and/or modify it under
+    the terms of the GNU Lesser General Public License (LGPL) as published
+    by the Free Software Foundation; either version 3 of the License, or
+    (at your option) any later version.  See <https://www.gnu.org/licenses/>.
+*/
+
+#include "test_helpers.h"
+#include "fq_default_poly.h"
+
+/// Given a fq_default_ctx, compute two random polynomials f1, f2 and a 
+/// random element a, and ensure f1(a) * f2(b) == (f1 * f2)(a).
```

Comment:
Commenting style

Reply:
Should multi line comments be of the form

```c
\*
  Text
  Here
*\
```

Reply:
I looked through the source code and tried to align the comment to what already exists

Reply:
I don't think we have any strong opinions here. Thanks!